### DISCUSSION TITLE
Is there a built in way to set passwords to expire after a set number of days?

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#14735] [Q&A] [answered]
Part of our policy is to force users to update their passwords after a period of time.  It seems like I'd have to alter the login in order to check before signing them in.  Before I do that,  I'd like to make sure there isn't a "preferred" way of doing this that wouldn't conflict with future updates.  I've searched the documentation and didn't come across anything concerning password expiration.

**Comment by hishamco** [comment, marked as answer]
I don't think there's a built-in functionality. FYI I'm planning to demo the same exact functionality and other auth features in the upcoming stand up meeting

**Reply by hishamco** [reply]
Basically, you can check the password expiration date with the current during the login and return an `IdentityResult` error once it expires
